README.md

```diff
@@ -88,6 +88,37 @@ or if you activated with a named path like `server`
 
     rake maintenance:server:end
 
+Redis
+=====
+
+Using the cli
+-------------
+__When your app is running__
+
+Start up ```redis-cli```
+
+__Use maintenance mode__
+
+```set turnout:maintenance <any value>```
+
+All the config should go in your config block in your app directory
+
+__Stop maintenance mode__
+
+```del turnout:maintenance```
+
+Using rake
+----------
+To activate, use
+
+```rake maintenance:start```
+
+To deactivate, use
+
+```rake maintenance:end```
+
+Redis mode currently does not support environmental variables
```

Comment:
Is there a reason not to support storing the settings in Redis? I'd think we could just write a JSON string to Redis or something  like that.

Reply:
I didn't know how :p
Also we didn't need the functionality at the time